tutorials/shading/visual_shader.rst

```diff
@@ -0,0 +1,101 @@
+.. _doc_visual_shader:
+
+Visual Shader
+=============
+
+Introduction
+------------
+
+VisualScript is an alternative for users that cannot code, Similarly, Visual Shader is the visual alternative for creating shaders.
+
+.. note::
+    
+   If you do not understand what shader is , go read :ref:`doc_what_are_shaders`
+   
+Creating a VisualShader
+-----------------------
+
+Create a new ``ShaderMaterial`` in some object of your choice.
```

Comment:
"VisualShaders can be created in any ShaderMaterial. To begin using VisualShaders, create a new ``ShaderMaterial`` in an object of your choice.